src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs

```diff
@@ -100,10 +100,14 @@ public void Configure(string name, OpenIdConnectOptions options)
         private async Task<OpenIdClientSettings> GetClientSettingsAsync()
         {
             var settings = await _clientService.GetSettingsAsync();
-            if ((await _clientService.ValidateSettingsAsync(settings)).Any(result => result != ValidationResult.Success))
-            {
-                _logger.LogWarning("The OpenID Connect module is not correctly configured.");
+            var errorResult = (await _clientService.ValidateSettingsAsync(settings)).Where(r => r != ValidationResult.Success);
 
+            if (errorResult.Any())
+            {
+                foreach (var item in errorResult)
+                {
+                    _logger.LogWarning(item.ErrorMessage);
+                }
```

Comment:
Just add a line after and it will looks good to me

Reply:
sorry，“ add a line after ”  after  where ? 
I don't quite understand your mean.

Reply:
Before the `return`, there's always should be line there in the code base